examples/example-react-vite/src/Components/Navigation/Navigation.tsx

```diff
@@ -72,6 +73,17 @@ export const Navigation: React.FC<Props> = ({ bridgeUrl }) => {
       icon: ICON_NAMES.GEAR,
       url: SEND_MESSAGE_URL,
     },
+    {
+      name: 'Send Wallet Action',
```

Comment:
Do we even want to expose this to dApps?  I was thinking dApps are limited to just the methods available on walletConnectService

Reply:
Yeah, I went around on that - it does provide a way for dapps to do things without waiting for specific support from us (i.e. specifically building `switchToGroup` in wc-js).

Reply:
Everything we add as a method should exist within walletConnectService first though, wouldn't it?  I don't think we'll ever have some hidden method that you can't call from the service.  Only benefit is not having to update the example/demo apps.